Assets/Scripts/Models/Furniture.cs

```diff
@@ -577,7 +577,7 @@ public void ReadXmlPrototype(XmlReader readerParent)
                     DragType = reader.ReadContentAsString();
                     break;
                 case "BuildingJob":
-                    ReadXMLBuildingJob(reader);
+                    ReadXmlBuildingJob(reader);
```

Comment:
It is. But for some reason it is Xml everywhere else, even the library uses Xml. So for consistency I changed it.